ArjunaJTA/narayana-jta-jakarta/pom.xml

```diff
@@ -1,136 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <groupId>org.jboss.narayana.jta</groupId>
     <artifactId>narayana-jta-all</artifactId>
     <version>5.12.2.Final-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <modelVersion>4.0.0</modelVersion>
   <artifactId>narayana-jta-jakarta</artifactId>
+  <packaging>jar</packaging>
   <name>Narayana: ArjunaJTA Jakarta narayana-jta-jakarta</name>
   <description>Narayana: ArjunaJTA Jakarta narayana-jta-jakarta (jta uber jar)</description>
-  <packaging>jar</packaging>
 
   <properties>
-    <version.jakarta.transaction.jakarta.transaction-api>2.0.0</version.jakarta.transaction.jakarta.transaction-api>
-    <version.jakarta.resource.jakarta.resource-api>2.0.0</version.jakarta.resource.jakarta.resource-api>
+    <maven.build.timestamp.format>yyyyMMdd'T'HHmmss</maven.build.timestamp.format>
+    <sortpom.skip>false</sortpom.skip>
   </properties>
-  
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.narayana.jta</groupId>
+      <artifactId>narayana-jta</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
   <build>
     <plugins>
       <plugin>
         <artifactId>maven-antrun-plugin</artifactId>
+        <dependencies>
+          <dependency>
+            <groupId>org.eclipse.transformer</groupId>
+            <artifactId>org.eclipse.transformer.cli</artifactId>
+            <version>0.2.0</version>
+          </dependency>
+          <dependency>
+            <groupId>ant-contrib</groupId>
+            <artifactId>ant-contrib</artifactId>
+            <version>1.0b3</version>
+            <exclusions>
+              <exclusion>
+                <groupId>ant</groupId>
+                <artifactId>ant</artifactId>
+              </exclusion>
+            </exclusions>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
             <id>transform-jar</id>
-            <phase>package</phase>
             <goals>
               <goal>run</goal>
             </goals>
+            <phase>package</phase>
             <configuration>
               <target>
-                <property name="plugin_classpath" refid="maven.plugin.classpath"/>
+                <property name="plugin_classpath" refid="maven.plugin.classpath"></property>
```

Comment:
I keep replacing these with just `/>` and they keep returning. I've done them all twice now.
I think there is a gremlin in the system and I do wonder if `sortpom` is really worthwhile using after all.

Reply:
I understand your pain! `sortpom` might sort too much sometime. On the other side, I support its use as it introduces (hidden) rules for contributors and keeps `pom` files nice and tidy.